src/styles/blocks/nav.scss

```
@@ -0,0 +1,68 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  &__items:not(:first-child) {
+    margin-left: 20px;
+  }
+
+  &__link {
+    display: flex;
+    text-align: center;
+    align-items: center;
+
+    position: relative;
+
+    font-family: Roboto, sans-serif;
```

```suggestion
    align-items: center;
    position: relative;
    font-family: Roboto, sans-serif;
```

not necessary to leave empty lines between properties